libs/core/src/lib/popover/base/base-popover.class.ts

```diff
@@ -33,6 +33,14 @@ export class BasePopoverClass {
     @Input()
     closeOnEscapeKey = true;
 
+    /** Whether to wrap content with fd-scrollbar directive. */
+    @Input()
+    disableScrollbar = false;
+
+    /** Should fd-scrollbar have tabindex*/
+    @Input()
+    tabbableScrollbar = true;
```

Comment:
After some investigation I think can avoid adding additional input to control this. 

When the focus inside the scrollable container user can scroll in any direction using Option\Alt + arrow keys, it's a default behavior provided by the browser by default.

Here is the example:  https://jsfiddle.net/7m6vb0td/18/

Reply:
@platon-rov 
_Option\Alt + arrow keys_ is not working for me in your example. BTW it's true that if any element inside `fd-scrollbar` have tabindex then it can be scrollable by keyboard. So it will work if either `div `with `fd-scrollbar` have tabindex or child within that have tabindex.